functionary/ui/templates/core/workflow_list.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base.html" %}
+{% block content %}
+    <div>
+        <div class="level-right">
+            <button class="button level-item is-link m-5"
+                    hx-target="body"
+                    hx-swap="beforeend"
+                    hx-get="{% url 'ui:workflow-create' %}">
+                <span class="icon is-small">
+                    <i class="fa fa-solid fa-plus"></i>
+                </span>
+                <span>Create</span>
+            </button>
+        </div>
+        <div class="table-container">
+            <table class="table is-hoverable is-fullwidth">
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Description</th>
+                        <th>Creator</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for workflow in object_list %}
+                        <tr>
+                            <td>
+                                <a href="{% url 'ui:workflow-detail' workflow.id %}">{{ workflow.name }}</a>
+                            </td>
+                            <td>{{ workflow.description }}</td>
+                            <td>{{ workflow.creator.username }}</td>
+                        </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+        </div>
+    </div>
+    {% include "partials/pagination_navbar.html" with page_obj=page_obj paginator=paginator %}
```

Comment:
This isn't your creation, but should the href on the button for the current page be disabled? It doesn't really make sense to click on the current page and have it load all over again.